UsageEnvironment/Log.cpp

```diff
@@ -141,8 +146,14 @@ void _Log::NextLogFile() {
 
 void _Log::BackupLog(const string backupFilePath) {
   lock_guard<mutex> backupLockGuard(logMutex);
+
+  if (!logFileDir.empty()) {
+    printf("Cannot backup live555 logs as it is not enabled.\n");
+    return;
+  }
+
   if (access(backupFilePath.c_str(), F_OK) == 0 && remove(backupFilePath.c_str()) < 0) {
-    fprintf(stderr, "Couldn't remove old backup file in %s%s\n", backupFilePath, strerror(errno));
+    fprintf(stderr, "Couldn't remove old backup file in %s%s\n", backupFilePath.c_str(), strerror(errno));
```

Comment:
Wouldn't `%s%s` result in the error being appended to the file path as one long continuous line of text?